source/checkpoint_restart.f90

```diff
@@ -390,7 +390,7 @@ subroutine crcheck
     write(crlog,"(a)") "CR_CHECK>  * Tracking variables"
     flush(crlog)
     read(cr_pntUnit(nPoint),iostat=ioStat) crnumlcr,crnuml,crsixrecs,crbinrec,cril,crnapxo, &
-      crnapx,cre0,crbetrel,crbrho,crnucmda
+      crnapx,cre0,crbeta0,crbrho,crnucmda
```

Comment:
I guess that the change from `betrel` to `beta0` is done elsewhere in the code, and only CR was missing, right?

Reply:
Not sure why this shows up as the variable is already `crbeta0` in master.

Reply:
This change also does not exist in the main diff.